.github/pull_request_template.md

```diff
@@ -1,6 +1,5 @@
 # Checklist
 
+* [ ] I read [CONTRIBUTING.md](../CONTRIBUTING.md).
 * [ ] Tests are added for new functionality or fixed bugs.
```

Comment:
```suggestion
* [ ] I added tests for the new functionality or fixed bugs.
```

Reply:
My idea was to use exactly the same wording there and in CONTRIBUTING.md to make it easier to copy&paste

Reply:
@AlekSi well, my concern is that for the first sentence you used active voice, and for the rest of the items - passive voice. So, there wasn't enough consistency there. Also, I believe, active voice is recommended when you implement a "call for action". Anyways, I leave it up to you.